README.md

```diff
@@ -3,44 +3,116 @@ Some helpful snippets of code to automate the creation of Cluster-Stacks
 
 ## Goals
 Creating cluster stacks and cluster based on these takes a significant
-number of stacks that are hard to remember correctly for people that are
-not cluster-API and Cluster Class expert.
+number of steps that are hard to remember correctly for people that are
+not cluster-API and Cluster Class experts.
 
-We this hide them in a number of distinct steps that are in numbered scripts.
-The reason for not doing everything in one script is that you do register
-cloud secrets or install capi much less often than install cluster classses
-which happens much less often than creating clusters.
+We hide them in a number of distinct scripts. The reason for not doing
+everything in one script is that you register cloud secrets or install CAPI
+much less often than install cluster classes, which happens much less often
+than creating clusters.
+
+## Prerequisites and Setup
+
+### Obtaining the Cloud-in-a-Box Certificate
+
+If you're using SCS Cloud-in-a-Box (CiaB), you'll need to obtain the certificate. The certificates are located on the CiaB manager host at `/etc/ssl/certs/ca-certificates.crt`.
+
+To obtain it:
+
+1. SSH into your CiaB manager host (usually as user `dragon`)
+2. Copy the certificate file:
+   ```bash
+   scp dragon@<CiaB-IP>:/etc/ssl/certs/ca-certificates.crt ./ca-certificates.crt
+   ```
+3. Extract the last certificate from the file (the CiaB-specific certificate):
+   ```bash
+   # Extract the last certificate from the bundle
+   tac ca-certificates.crt | awk '/-----BEGIN CERTIFICATE-----/{flag=1} flag{print} /-----END CERTIFICATE-----/{exit}' | tac > ciab.cert
+   ```
+4. Place this `ciab.cert` file in the same directory as your scripts
+
+### Preparing Configuration Files
+
+Before running the main scripts, you may want to use the utility script to prepare your configuration:
+
+1. Place your `ciab.cert` file (or any other OpenStack certificate with `.cert` extension) in the script directory
+2. Create your `clouds.yaml` file from the template
+3. Run the preparation script:
+   ```bash
+   ./99-prepare-files.sh
+   ```
+
+This script will:
+- Create the `~/.config/openstack` directory if it doesn't exist
+- Copy any `.cert` files to the config directory
+- Update `clouds.yaml` to reference the certificate with its absolute path
+- Copy the updated `clouds.yaml` to the config directory
 
 ## Settings
 There is a [cluster-settings-template.env](cluster-settings-template.env) file
 that contains the parameters typically adjusted by users. Please create a
-copy, fill it in, and pass it to the scripts.
+copy, fill it in, and pass it to the scripts. You can also use the 
+[cluster-settings.env.sample](cluster-settings.env.sample) file as a reference, 
+which includes example values for all parameters.
+
+### Configuration Parameters
+
+#### Registry and Repository Settings
```

Comment:
I like the way you structure the settings in different groups.
We probably should be thinking about new prefixes for the first two categories.
We also need to make sure we don't duplicate information too much, as divergence is otherwise hard to avoid.
We have the main documentation for the parameters in the -template.env file. It would be my preferred place, TBH.
Additional documentation then should NOT repeat the explanations in there, but introduce higher level concepts etc.